openfisca_web_api_preview/app.py

```diff
@@ -81,6 +98,8 @@ def get_variable(id):
 
     @app.route('/spec')
     def get_spec():
+        if data['host'] is None:
```

Comment:
This condition is checked in `def resolve_host`. Isn't it redundant to have it here also ?

Reply:
The `spec` route can be called before the `/` one. In that case, `data['host']` is still `None` when we get here, so we need to resolve the host.

Reply:
However, I was just told that re-using the host from one request to another was risky (one local request, and everyone is seeing a `localhost:3000` as a host) and not very stateless, so we need a different approach 😞

Reply:
Ok, adapted the code :)